hw/ip/kmac/dv/cov/kmac_masked_cov_excl.el

```diff
@@ -0,0 +1,52 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: udij
+// Format Version: 2
+// Date: Wed Jun 16 17:36:29 2021
+// ExclMode: default
+//==================================================
+CHECKSUM: "2274358225"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
+INSTANCE:tb.dut.u_tlul_adapter_msgfifo.u_rsp_gen
+CHECKSUM: "2274358225"
+ANNOTATION: "[LOW_RISK] discussed with designer, collecting coverage on adapter SRAM and all internal submodules is considered to be unimportant"
```

Comment:
@eunchan 

Just want to ensure again we really want to do this. `tlul_adapter` is not verified at its own level. We decided to close DV on all prims and tl building blocks at the IP level.

Reply:
If that's the case, I think we at least should have FPV test for the adapter. Do we have?

Reply:
And. I agree it is good to exclude prims/ and components from the IP level DV and having those coverages somewhere else.

Reply:
CC: @tjaychen @msfschaffner

Reply:
Prims / tl building blocks are significantly parameterized. Having DV testbench at that level is out of question. Even with FPV, it will not be sufficient to only verify the default parameter set (I believe that is what we do atm). There is no mechanism in place to know ALL possible parameter sets used in ALL IPs for all prims / tl building blocks. That's the reason we chose to close coverage at the IP level instead.

Reply:
In contrary, If we check the coverage in the IP level for highly parametrizable IP, the stimulus most likely not cover the every corner case (e.g SRAM adapter, may the request completed in a cycle so no way to hit the full condition). Then, the DV engineer and the designer need to review the coverage report for the prim component in detail in every IP.

BTW, I mis-read your first comment :) sorry

Reply:
That's a small price to pay for the peace of mind :)

Reply:
I don't think that is small price at all. If the IP designer knows the primitives and components well, then yes, it won't take long to review. If the designer only knows the interface of the prims/components, then it is whole different story.

I think it would be better to prepare the FPV and give some compile options for the parameters, so that designers/ dv engineers to run the FPV with given parameters. The parameters could be derived from the IP level.

Reply:
@eunchan @sriyerg I think we should make forward progress on this PR - shall we remove the exclusions of the `tlul_adapter_msgfifo` and keep this discussion for a V3 timeframe?
Most of the module is covered to begin with so IIRC it's not too big of a coverage impact (can always revisit this topic if it is somehow necessary to hit 90% coverage for V2).

Reply:
Yup. I am ok with it. Waive and create an issue for discussing the item.